frontend/src/metabase/ui/components/utils/Portal/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Portal } from "@mantine/core";
```

Comment:
@ranquild would you be ok with adding Portal to `metabase/ui`?

Reply:
I'm okay with this